ui/widgets/nuxeo-user-tag.js

```diff
@@ -169,6 +186,18 @@ import './nuxeo-tooltip.js';
     _preventPropagation(e) {
       e.stopPropagation();
     }
+
+    _computeLabel(user) {
+      return this._name(user);
+    }
+
+    _ariaLabelChanged(newValue, oldValue) {
+      const ariaLabel = this.getAttribute('aria-label');
+      // Don't stomp over a user-set aria-label.
+      if (!ariaLabel || oldValue === ariaLabel) {
+        this.setAttribute('aria-label', newValue);
+      }
+    }
```

Comment:
I'm a bit confused by this. Shouldn't we do the check when the user changes, instead? When the user changes, if there is no arial-label defined, we should defined one with the user info. When the ariaLabel changes, how do we know who triggered it?

Reply:
As I think about this, check on the user observer would have the same problem ,as it would not know who set it before: was it set from the outside or was it set by the element?